develop/networks.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Networks
+description: Explore the Polkadot ecosystem networks and learn the unique purposes of each, tailored for blockchain innovation, testing, and enterprise-grade solutions.
+---
+
+# Networks
+
+## Introduction
+
+The Polkadot ecosystem consists of multiple networks designed to support different stages of blockchain development, from main networks to test networks. Each network serves a unique purpose, providing developers with flexible environments for building, testing, and deploying blockchain applications.
+
+## Main Networks
+
+### Polkadot
+
+- **Description** - primary production blockchain network for high-stakes, enterprise-grade applications
+- **RPC URL** - `wss://polkadot-rpc.dwellir.com`
```

Comment:
Ideally all of these are wrapped in 3 backticks so users can easily copy the RPC endpoints

Reply:
Discussed via slack, but adding it here for posterity, we should try to provide a list of RPC providers / endpoints. Maybe we can format this in a way that makes the three back ticks look less weird

Reply:
We have added RPC URLs to tables with backticks, so that's an option. You can view an example [here](https://github.com/moonbeam-foundation/moonbeam-docs/blob/master/.snippets/text/builders/get-started/endpoints/moonbase.md?plain=1) of how to do that